basilisk/tests/test_main_app.py

```
@@ -0,0 +1,280 @@
+import pytest
+import unittest.mock as mock
```

Ruff (0.11.9)

`unittest.mock` imported but unused

Remove unused import: `unittest.mock`

(F401)